target/posix/CMakeLists.txt

```diff
@@ -129,5 +129,6 @@ if (CONFIG_MENDER_PLATFORM_TLS_TYPE STREQUAL "mbedtls")
 endif()
 
 # Define version
-file(STRINGS ${MENDER_MCU_VERSION_FILE} MENDER_CLIENT_VERSION)
-add_definitions("-DMENDER_CLIENT_VERSION=\"${MENDER_CLIENT_VERSION}\"")
+include("${MENDER_MCU_ROOT}/cmake/git.cmake")
```

Comment:
I'd actually name the file differently because it only covers `git describe`. But not a biggie.

Reply:
Yeah, but then I'd vote for `git_helpers.cmake` because `git.cmake` looks to me like the base one that `find_package(Git)` should find and load. But YMMV.